integration/auth/user_mgt_test.go

```diff
@@ -52,19 +64,33 @@ func TestUserManagement(t *testing.T) {
 func testCreateUsers(t *testing.T) {
 	// Create users with uid
 	for i := 0; i < 3; i++ {
-		params := (&auth.UserToCreate{}).UID(fmt.Sprintf("tempTestUserID-%d", i))
+		uid := fmt.Sprintf("tempTestUserID-%d", i)
+		params := (&auth.UserToCreate{}).UID(uid)
 		u, err := client.CreateUser(context.Background(), params)
 		if err != nil {
-			t.Fatal("failed to create user", i, err)
+			if strings.Contains(err.Error(), "DUPLICATE_LOCAL_ID") {
+				u, err = client.GetUser(context.Background(), uid)
+				if err != nil {
+					t.Fatal(err)
+				}
+			} else {
+				t.Fatal(err)
+			}
 		}
 		testFixtures.uidList = append(testFixtures.uidList, u.UID)
 	}
-
 	// Create user with no parameters (zero-value)
 	u, err := client.CreateUser(context.Background(), (&auth.UserToCreate{}))
 	if err != nil {
 		t.Fatal(err)
 	}
+	// make sure that the user.TokensValidAfterTime is not in the future or stale.
+	if u.TokensValidAfterTime > time.Now().Unix()*1000 {
+		t.Errorf("timestamp cannot be in the future")
+	}
+	if u.TokensValidAfterTime < (time.Now().Unix()-3600)*1000 {
```

Comment:
If you want: `time.Now().Add(time.Duration(-1) * time.Hour)` will do the date math.

Reply:
Thanks! (-time.Hour  also works, but i changed it around a bit. )